internal/operators/ocs/validations.go

```diff
@@ -44,35 +27,31 @@ type Config struct {
 	OCSDeploymentType              string `envconfig:"OCS_DEPLOYMENT_TYPE" default:"None"`
 }
 
-// ValidateOCSRequirements is used to validate min requirements of OCS
-func (o *ocsValidator) ValidateRequirements(cluster *models.Cluster) (api.ValidationStatus, string) {
+// validateRequirements is used to validate min requirements of OCS
+func (o *operator) validateRequirements(cluster *models.Cluster) (api.ValidationStatus, string) {
 	var status string
-	var err error
-	var inventoryMissing bool
 	hosts := cluster.Hosts
 
-	if int64(len(hosts)) < o.OCSRequiredHosts {
+	//Check if number of hosts is less than required for compact mode
+	if int64(len(hosts)) < o.config.OCSRequiredHosts {
 		status = "Insufficient hosts to deploy OCS. A minimum of 3 hosts is required to deploy OCS. "
 		o.log.Info("Validate Requirements status ", status)
```

Comment:
It would translate to `Validate Requirements status Insufficient hosts to deploy OCS. A minimum of 3 hosts is required to deploy OCS. ` it does contain OCS, should we change it to a better message?